src/store/index.js

```diff
@@ -1,6 +1,9 @@
 import { configureStore } from "@reduxjs/toolkit";
+import weatherSlice from "./features/weatherSlice";
```

Comment:
Let's rename `weatherSlice` as `weatherReducer` because we exported the slice's reducer with `export default weatherSlice.reducer;` code.